marlowe-dashboard-server/src/Marlowe/Run/Webserver/Server.hs

```diff
@@ -0,0 +1,206 @@
+import qualified Cardano.Wallet.Api.Types                   as WBE
+import           Marlowe.Run.Webserver.API                  (API)
+
+import           Cardano.Wallet.Api                         (WalletKeys)
+import           Cardano.Wallet.Mock.Types                  (WalletInfo (..))
+import qualified Cardano.Wallet.Primitive.AddressDerivation as WBE
+import qualified Cardano.Wallet.Primitive.Types             as WBE
+
+import           Control.Monad.Except                       (ExceptT (ExceptT), runExceptT, withExceptT)
+import           Control.Monad.IO.Class                     (MonadIO, liftIO)
+import           Control.Monad.Reader                       (ReaderT, runReaderT)
+import           Data.Aeson                                 as Aeson
+import qualified Data.Aeson.Types                           as Aeson
+
+import           Cardano.Wallet.Primitive.AddressDerivation (Passphrase (Passphrase))
+import           Control.Monad.Reader.Class
+import qualified Data.ByteString.Lazy                       as BL
+import           Data.Proxy                                 (Proxy (Proxy))
+import           GHC.Generics                               (Generic)
+
+import           Data.String                                as S
+import           Data.Text                                  (Text)
+import qualified Data.Text                                  as Text
+import           Data.Text.Class                            (FromText (..))
+import           Ledger                                     (PubKeyHash (..))
+import           Marlowe.Run.Webserver.Types                (Env, RestoreError (..), RestorePostData (..))
+import qualified Marlowe.Run.Webserver.WebSocket            as WS
+import           PlutusTx.Builtins.Internal                 (BuiltinByteString (..))
+import           Servant                                    (Handler (Handler), Server, ServerError, hoistServer,
+                                                             serveDirectoryFileServer, (:<|>) ((:<|>)), (:>))
+import           Servant.Client                             (ClientError (FailureResponse), ClientM,
+                                                             ResponseF (responseBody), client, runClientM)
+import           Text.Regex                                 (Regex)
+import qualified Text.Regex                                 as Regex
+import qualified Wallet.Emulator.Wallet                     as Pab.Wallet
+
+handlers :: FilePath -> Env -> Server API
+handlers staticPath env =
+    hoistServer (Proxy @API) liftHandler
+        ( WS.handle
+            :<|> (handleVersion
+                   :<|> restoreWallet
+                 )
+            :<|> serveDirectoryFileServer staticPath
+        )
+    where
+    liftHandler :: ReaderT Env (ExceptT ServerError IO) a -> Handler a
+    liftHandler = Handler . flip runReaderT env
+
+-- TODO: Can we get this from cabal somehow?
```

Comment:
@shlevy how would you solve this issue? Should we put this version on a ENV variable?

Reply:
`Paths_pkgname` includes a version export: https://cabal.readthedocs.io/en/3.6/cabal-package.html#accessing-data-files-from-package-code

Reply:
You can also use [`Cardano.Config.Git.Rev.gitRev`](https://github.com/input-output-hk/cardano-config/blob/master/src/Cardano/Config/Git/Rev.hs#L16) if you want the git revision.